pages/index.js

```diff
@@ -16,67 +21,152 @@ export default function Home() {
       <Heading as="h1" variant="heading10">
         Welcome to the Paste Prototyping Kit
       </Heading>
+      <Separator orientation="horizontal" verticalSpacing="space50" />
+      <Heading as="h2" variant="heading20">
+        Getting Started
+      </Heading>
+      <Paragraph>
+        Build your first prototype, an <strong>&quot;About Me&quot;</strong> page, by following our step-by-step guide.
+      </Paragraph>
+      <Box marginBottom="space40">
+        <Alert variant="warning">
+          <Text>
+            <strong>Only edit files in the pages folder.</strong> It contains everything necessary to build your
```

Comment:
I think I did that too everywhere else but here! I wasn't sure if it would be possible to use Code inside of 'strong' text. I'll check it out and get back to you

Reply:
![Screen Shot 2021-02-02 at 1 51 53 PM](https://user-images.githubusercontent.com/77117846/106667475-d1bb2980-655d-11eb-8f88-bcee790b0fbf.png)
I tried using Code for it but it looks... weird? I think I'll just put it in single quotes like this:
![Screen Shot 2021-02-02 at 1 54 43 PM](https://user-images.githubusercontent.com/77117846/106667770-37a7b100-655e-11eb-8585-d47b6cd16d73.png)
Thoughts?

Reply:
That looks good!! @olipyskoty and I used double quotes on delete.js so we can change to single.

Reply:
@nkrantz perfect, thank you!